src/pages/service-mesh-management/meshery/getting-started.js

```diff
@@ -1,18 +1,18 @@
 import React from "react";
```

Comment:
@leecalcote Can you avoid changing this Page. I will go it #1138 as it will lead to conflict :grimacing:

Reply:
Sure. I can merge changes there, too. Did you mean to point to a different PR / issue #?

Reply:
Yeah I meant to point to #1338. If you can merge into that branch or revert the changes in this file. Both works

Reply:
Thanks for taking care of this, @Tanuj22. I missed your last comment until just now. 😬